interface_frontend.py

```diff
@@ -427,3 +429,63 @@ def handle_text(interface_state, text):
     # Confirm selection of cards
     if text == 'k':
         interface_state.confirm_selection()
+
+
+def handle_click(interface_state, x, y, window):
+    """
+    Interface react on mouse press.
+    """
+    # Transform x, y coordinates according to zoom of window.
+    zoom = min(
+        window.height / WINDOW_HEIGHT,
+        window.width / WINDOW_WIDTH
+    )
+    x, y = (x / zoom, y / zoom)
+    # Select a card and take it in your "hand"
+    # Selected card is in "GREEN" cursor
+    card_sprite = cards_type_sprites["u_turn"]
+    for i, coordinate in enumerate(dealt_cards_coordinates):
+        coord_x, coord_y = coordinate
+        if coords_in_rect(
+            x, y, coord_x, coord_y, card_sprite.width, card_sprite.height
+        ):
+            interface_state.select_card(i)
+
+    # Confirm selection of cards
+    if coords_in_rect(
+        x, y, indicator_red_sprite.x, indicator_red_sprite.y,
+        indicator_red_sprite.width, indicator_red_sprite.height
+    ):
+        interface_state.confirm_selection()
+
+    # Put and take a Power Down token
```

Comment:
```suggestion
    # Switch on and off Power Down token
```
_put_ and _take_ popisuji spis tu akci pri hrani tradicni deskovky, u nas je to vic "zapnout", "vypnout".

Reply:
já to jen zkopírovala jak je to v on_text, opravím i tam :)